docs/HACKING.md

```diff
@@ -54,6 +82,121 @@ There are cases where determining a type for a particular object can be difficul
 
 **Note: You will need access to a running instance of MAAS in order to run maas-ui.**
 
+## Run MAAS-UI on your local machine
+
+You can run MAAS-UI on your local machine assuming that you already have an instance of MAAS running somewhere that you can connect to.
+
+In the following sections we assume that you're having your MAAS back-end running on `http:/10.10.0.30:5240`. This can easily be adapted to other IPs, names, or `https`.
```

Comment:
double `/` after "http"

```suggestion
In the following sections we assume that you're having your MAAS back-end running on `http://10.10.0.30:5240`. This can easily be adapted to other IPs, names, or `https`.
```